locations/spiders/dino_pl.py

```diff
@@ -25,8 +29,11 @@ def parse_decryption_params(self, response):
             key = m.group(1)
         if m := re.search(r"""\.from\([\n ]*['"]([0-9a-f]{32})['"],[\n ]*['"]hex['"][\n ]*\)""", response.text):
             iv = m.group(1)
+        if m := re.search(r"a[\s=]+\"(.+?)\",", response.text):
```

Comment:
Could this regex be a bit more robust to prevent future false positives? The variable `a` may not be guaranteed, for example.

Perhaps the following instead:
`r"""[a-z]\s*=\s*"([0-9a-f]{40}])""""`

(e.g. capture the value of a variable where the value is a base64 encoded token 40 characters long)

Reply:
Changes applied. Thanks.